.github/workflows/validate-pr-title.yml

```diff
@@ -0,0 +1,24 @@
+name: Validate PR Title
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+    branches:
+      - feature/not-dev
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+
+      - name: Validate PR title
```

Comment:
I think it would be very useful to see what the entry will be in the changelog when you create/update the pr.
Just to sanity check what will be added.
Could we have the validation also output what the log entry will be to stdout, so if you are interested you can check the output of the action?

Reply:
Fixed